media/js/firefox/new/experiment-cliqz-funnelcakes.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function(Mozilla) {
+    'use strict';
+
+    // initialize experiment - performs env check and does geo-lookup
+    Mozilla.TCFCGeoExp.init({
+        countryCode: 'us',
```

Comment:
Should this be 'de'?

Reply:
Yeah, this has just been left in for testing purposes